task/templates/task.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html' %}
+{% block content %}
+    <h2> Tasks </h2>
+    <ul>
+        {% for task in tasks %}
+            <li>
+             {% if task.2 == 6 %}
```

Comment:
jak pozbędziesz się values_list to tutaj nie trzeba będzie odwoływac się po indeksach